suite-native/link/package.json

```diff
@@ -10,8 +10,11 @@
         "type-check": "tsc --build"
     },
     "dependencies": {
+        "@suite-native/atoms": "workspace:*",
```

Comment:
Please remove these dependendencies from this package, they seems unused

Reply:
[1d456aa](https://github.com/trezor/trezor-suite/pull/7889/commits/1d456aae554fa46269c007c6047352dfb9d28407)